src/ducks/GainersAndLosers/GainersAndLosersTable.js

```diff
@@ -1,69 +1,63 @@
 import React from 'react'
 import ReactTable from 'react-table'
-import 'react-table/react-table.css'
+import moment from 'moment'
+import { graphql } from 'react-apollo'
 import getColumns from './gainers-and-losers-table-columns'
-import { fetchGainersOrLosers } from './actions'
-import { connect } from 'react-redux'
-import { bindActionCreators } from 'redux'
+import { TOP_SOCIAL_GAINERS_LOSERS_GQL } from './gainersLosersGQL'
```

Comment:
Instead of TOP_SOCIAL_GAINERS_LOSERS_GQL
TOP_SOCIAL_GAINERS_LOSERS_QUERY as Apollo team does. We make it for all new code.